core/src/main/scala/cats/MonadTrans.scala

```diff
@@ -0,0 +1,17 @@
+package cats
+
+/**
+ * A type class which abstracts over the ability to lift an M[A] into a
+ * MonadTransformer
+ */
+trait MonadTrans[MT[_[_], _]] {
+
+  /**
+   * Lift a value of type M[A] into a monad transformer MT[M, A]
+   */
+  def liftT[M[_]: Monad, A](ma: M[A]): MT[M, A]
```

Comment:
A simple way would be to pick a fixed `M`, e.g. `Option`, get the `Monad[MT[Option, ?]]` from this method, and then law-check that.

A fancier way would be to use existentials or something to be able to vary `M` and do law-checking across all these :-)